pytorch_lightning/core/lightning.py

```diff
@@ -1022,7 +1022,8 @@ def load_from_checkpoint(cls, checkpoint_path):
 
     def summarize(self, mode):
         model_summary = ModelSummary(self, mode=mode)
-        logging.info(model_summary)
+        if logging.getLogger().level <= logging.INFO:
```

Comment:
I would not remove the logging, then make it as a switch:
```python
show = print if logging.getLogger().level <= logging.INFO else logging.info
show(model_summary)
```

Reply:
Well, we need to remove the prepending `INFO` when using logging.

As I understand, `logging.getLogger().level <= logging.INFO` only true when `INFO` and `DEBUG` ?

What's the purpose of using logging.info instead of print when level is bigger then INFO?

Reply:
the logging levels are here: https://docs.python.org/3/library/logging.html#logging-levels

Reply:
you can have different level to write logs to a file and this removing `logging.info(model_summary)` will skip it completely... so at least keep it there

Reply:
Okay I get it, lets bring it back.

After a little digging, it seems a bit hard to remove the prepending `INFO` :| , I shall leave this to some other day.

Reply:
What is the entry point, usually it is enough to set `logging.basicConfig(level=logging.INFO)`

Reply:
It was set actually 
https://github.com/williamFalcon/pytorch-lightning/blob/a6d64ac013868160fafdda6a0794dc6709301967/pytorch_lightning/trainer/trainer.py#L244

Reply:
but it is probably too low, it has to be done at the beginning of a script, in main...
usually, the config does not have any effect since logging is used for the first time

Reply:
I was also confused why the baseConfig line does not set the info level globally. The INFO in front messes up the summary, but we could add a new line at the beginning of the summary, maybe?

Reply:
I would consider to add it to the package `__init__`

Reply:
That might be better. Honestly I didn't know what is the beginning of the package, so I left it in Trainer

Reply:
`core/__init__` or even `pytorch_ligthning/__init__`

Reply:
Good catch @Borda. Of course, this means we will still have the [INFO] issue for logged files :) But this will be useful for everyone.

Reply:
you can move the config to `pytorch_ligthning/__init__` then